app/assets/javascripts/discourse/app/components/tap-tile.js

```diff
@@ -5,25 +5,25 @@ import { propertyEqual } from "discourse/lib/computed";
 export default Component.extend({
   init() {
     this._super(...arguments);
-
     this.set("elementId", `tap_tile_${this.tileId}`);
   },
 
   classNames: ["tap-tile"],
-
   classNameBindings: ["active"],
-
   attributeBindings: ["role", "ariaPressed", "tabIndex"],
-
   role: "button",
-
   tabIndex: 0,
-
   ariaPressed: reads("active"),
 
   click() {
     this.onChange(this.tileId);
   },
 
+  keyDown(e) {
+    if (e.which === 13) {
```

Comment:
Minor suggestion but I think assigning the number to a constant can help to improve readability.

Reply:
Sure, we can actually do 
```suggestion
    if (e.key === "Enter") {
```